Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -50,3 +50,7 @@ $VFS_SCRIPTDIR/PublishGit.sh $GITPATH || exit 1
 
 echo 'Running VFS for Git unit tests...'
 $VFS_PUBLISHDIR/GVFS.UnitTests || exit 1
+
+echo 'Creating installer package...'
```

Comment:
I think this should remain a separate step for now. For your developer inner loop scenarios, we don't need to build the installer each time.